site/en/codelabs/openthread-border-router-ipv6-multicast/index.lab.md

```diff
@@ -92,24 +92,24 @@ Done
 ```
 
 > aside positive
-> 
-> Dive Deeper: 
-> 
+>
+> Dive Deeper:
```

Comment:
The earlier Dive Deeper is bold, but this one is not.  They should be consistent, so one should change (up to you)